pkg/config/model.go

```diff
@@ -390,3 +398,15 @@ func (g GroupVersionKind) CanonicalGroup() string {
 // PatchFunc provides the cached config as a base for modification. Only diff the between the cfg
 // parameter and the returned Config will be applied.
 type PatchFunc func(cfg Config) (Config, kubetypes.PatchType)
+
+type Namer interface {
+	GetName() string
+	GetNamespace() string
+}
+
+func Name(n Namer) kubetypes.NamespacedName {
```

Comment:
how about NamespacedName

Reply:
I am not sure what you mean. Instead of Namer?

Reply:
I mean rename the function name